app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
```

Comment:
Toán tử này nghĩa là gì em
 ||=

Reply:
- toán tử ||= này để gán cho biến @current_user 1 giá trị id
- nếu lúc đầu @current_user = nil thì sau khi sử dụng 
              @current_user ||= User.find_by(id: session[:user_id])
    thì @current_user sẽ bằng giá trị của     User.find_by(id: session[:user_id]) trả về.
 nếu @current_user != nil hoặc != false thì bằng chính nó.